databend-jdbc/src/main/java/com/databend/jdbc/AbstractDatabendResultSet.java

```diff
@@ -562,8 +562,8 @@ public Timestamp getTimestamp(int columnIndex)
     private Timestamp getTimestamp(int columnIndex, DateTimeZone localTimeZone)
             throws SQLException {
         Object value = column(columnIndex);
-        if (value == null) {
-            return null;
+        if (value == null || value.toString().equalsIgnoreCase("null")) {
```

Comment:
If others data type is NULL, need to handle too?

Reply:
> If others data type is NULL, need to handle too?

Other data types seems handle correctly.